source/getting-started/quick-start.md

```diff
@@ -0,0 +1,262 @@
+the background. You should see that "Welcome to Ember" has been
+replaced by "PeopleTracker".
+
+## Define a Route
+
+Let's build an application that shows a list of scientists. To do that,
+the first step is to create a route. For now, you can think of routes as
+being the different pages that make up your application.
+
+Ember comes with _generators_ that automate the boilerplate code for
+common tasks. To generate a route, type this in your terminal:
+
+```sh
+ember generate route scientists
+```
+
+You'll see output like this:
+
+```text
+installing route
+  create app/routes/scientists.js
+  create app/templates/scientists.hbs
+updating router
+  add route scientists
+installing route-test
+  create tests/unit/routes/scientists-test.js
+```
+
+That's Ember telling you that it has created:
+
+1. A template to be displayed when the user visits `/scientists`
+2. A `Route` object that fetches the model used by that template
+3. An entry in the application's router (located in `app/router.js`)
+4. A unit test for this route
+
+Open the newly-created template in `app/templates/scientists.hbs` and add
+the following HTML:
+
+```app/templates/scientists.hbs
+<h2>List of Scientists</h2>
+```
+
+In your browser, open
+[http://localhost:4200/scientists](http://localhost:4200/scientists). You should
+see the `<h2>` you put in the `scientists.hbs` template, right below the
+`<h1>` from our `application.hbs` template.
+
+Now that we've got the `scientists` template rendering, let's give it some
+data to render. We do that by specifying a _model_ for that route, and
+we can specify a model by editing `app/routes/scientists.js`.
+
+We'll take the code created for us by the generator and add a `model()`
+method to the `Route`:
+
+```app/routes/scientists.js{+4,+5,+6}
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  model() {
+    return ['Marie Curie', 'Mae Jemison', 'Albert Hofmann'];
+  }
+});
+```
+
+(This code example uses the latest features in JavaScript, some of which
+you may not be familiar with. Learn more with this [overview of the
+newest JavaScript features][es6-bullet-points].)
+
+[es6-bullet-points]: https://ponyfoo.com/articles/es6
+
+In a route's `model()` method, you return whatever data you want to make
+available to the template. If you need to fetch data asynchronously, the
+`model()` method supports any library that uses [JavaScript
+Promises][promises].
+
+[promises]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise
+
+Now let's tell Ember how to turn that array of strings into HTML. Open
+the `scientists` template and add some Handlebars code to loop through the
+array and print it:
+
+```app/templates/scientists.hbs{+3,+4,+5,+6,+7}
+<h2>List of Scientists</h2>
+
+<ul>
+  {{#each model as |scientist|}}
+    <li>{{scientist}}</li>
+  {{/each}}
+<ul>
+```
+
+Here, we use the `each` helper to loop over each item in the array we
```

Comment:
Stuff like this should link to the relevant docs to provide confidence to the reader that they'll be able to chase down that learning thread if/when they need to.